dev-util/rt-tests/rt-tests-1.5.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python3_{6,7,8})
+
+inherit python-single-r1
+
+DESCRIPTION="A collection of latency testing tools for the linux(-rt) kernel"
+HOMEPAGE="https://git.kernel.org/pub/scm/utils/rt-tests/rt-tests.git/about/"
+SRC_URI="https://kernel.org/pub/linux/utils/rt-tests/${P}.tar.xz"
```

Comment:
I tried to test this, but seems like they dropped 1.5 altogether. Buggy release? Please update.

Reply:
It seems like that v1.6 has been releases in the meantime. I've updated it.
The old releases are still available on the server in the older/ directory.